### ISSUE TITLE
cant resolve dependency tree

### ISSUE THREAD

**Issue by rod-dd** [rejvban/nestjs-twilio#11]
# Bug Report
### NestJS Info
![image](https://user-images.githubusercontent.com/85271039/120627833-a5c19d80-c496-11eb-838d-6808bc1c5585.png)

## Describe the Bug
installing this npm causes an error
```sh
npm i nestjs-twilio 
```
and give me this
![image](https://user-images.githubusercontent.com/85271039/120628015-d0abf180-c496-11eb-9775-42912a9f457d.png)

## Expected Behavior
it should install properly or without an error.

## Workaround
you can install the npm by this,
```sh
npm i nestjs-twilio  --legacy-peer-deeps
```
Ill update this ticket if I find something meaningful.

**Comment by lkaric** [comment]
@rod-dd I bumped the versions and cleaned up some stuff, can you try installing now with the latest [` v1.2.0`](https://github.com/rejvban/nestjs-twilio/releases/tag/v1.2.0), please let me know if it fixed the issue

**Comment by rod-dd** [comment]
@rejvban thanks man,

Managed to quick install under my project and this is what i got
![image](https://user-images.githubusercontent.com/85271039/120908867-5e5b2d00-c6a1-11eb-8936-77f4e0032716.png)

I think this "constrain" is due to this line: https://github.com/rejvban/nestjs-twilio/blob/master/package.json#L66.

Let me hear your thoughts.

**Comment by lkaric** [comment]
I released a v1.2.1 which hopefully supports the whole 7 major version

**Comment by rod-dd** [comment]
Yay! thanks man @rejvban, managed to install without an issue.

![image](https://user-images.githubusercontent.com/85271039/120947003-f96f0800-c770-11eb-92c6-fbe14ae1ebac.png)
